src/Annotation/ElementAnnotationsListener.php

```diff
@@ -315,18 +317,16 @@ public function handleInputAnnotation($e)
     }
 
     /**
-     * Handle the Object and Instance annotations
+     * Handle the Instance annotations
      *
      * Sets the object to bind to the form or fieldset
      *
      * @param  EventInterface $e
      * @return void
      */
-    public function handleObjectAnnotation($e)
+    public function handleInstanceAnnotation($e)
```

Comment:
Type declaration and return type is missing, DocBlock can also be reduced.

Reply:
They are in place as DocBlock, as you see.
I'd like to have a massive cleanup for `3.0.0` for all these things, including these ones.

Is it ok for you to have this change in a separate PR so we have smaller diff here?

Reply:
@Slamdunk If desired, I can provide a PR for such things once the open MRs have been merged.